models/notification.go

```diff
@@ -0,0 +1,134 @@
+package models
+
+import (
+	"time"
+)
+
+const (
+	NotificationStatusUnread = "U"
+	NotificationStatusRead   = "R"
+
+	NotificationSourceIssue       = "I"
+	NotificationSourcePullRequest = "P"
+	NotificationSourceCommit      = "C"
+)
+
+type Notification struct {
+	ID     int64 `xorm:"pk autoincr"`
+	UserID int64 `xorm:"INDEX NOT NULL"`
+	RepoID int64 `xorm:"INDEX NOT NULL"`
+
+	Status string `xorm:"VARCHAR(1) INDEX NOT NULL"`
+	Source string `xorm:"VARCHAR(1) INDEX NOT NULL"`
```

Comment:
These should be `type NotificationStatus string` and `type NotificationSource string` (if string, see above comment)